src/json-ld.net/Core/JsonLdOptions.cs

```diff
@@ -43,6 +42,9 @@ public virtual JsonLD.Core.JsonLdOptions Clone()
 
         private bool produceGeneralizedRdf = false;
 
+        private bool sortGraphsFromRdf = true;
```

Comment:
Since I've only been able to control the order with the `FromRDF` method (which luckily is all that's needed to get the desired behavior), I made the option pretty narrow in scope.